azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_kubernetes.go

```diff
@@ -88,11 +88,14 @@ func makeSchemaKubeconfig(r *schema.Resource) {
 			"kube_config": {
 				Type:        schema.TypeString,
 				Required:    true,
+				Sensitive:   true,
+				DefaultFunc: schema.EnvDefaultFunc("AZDO_KUBERNETES_SERVICE_CONNECTION_KUBECONFIG", nil),
```

Comment:
@mouellet  Is there any reason why remove the `DefaultFunc`? This will be a breaking change to some users if they set the `k8s_condfig` form env configuration.

Reply:
Hi @xuzhang3,

Thanks for the review! 

As you can see on line 92, I made sure the `DefaultFunc` is still there. It was previously set through the `makeProtectedSchema` function and now it's set directly on the `kube_config` schema element.